demo/src/API.elm

```diff
@@ -30,11 +30,11 @@ main =
         , initialModel = Model []
         , parseRoute = UrlParser.parseString Route.route
         , update = update
-        , subscriptions = subscriptions
+        , interop = Serverless.Interop interopEncode interopDecoder
```

Comment:
Replacing general purpose `subscriptions` with `interop` parameter. This is because we need to associate calls to JavaScript with specific connections. `Serverless.Interop` provides a way to do that